modules/verify-central-install-operator.adoc

```diff
@@ -12,7 +12,24 @@ After Central finishes installing, log in to the RHACS portal to verify the succ
 . Select the {product-title} Operator from the list of installed Operators.
 . Select the *Central* tab.
 . From the *Centrals* list, select `stackrox-central-services` to view its details.
-. The *Admin Credentials Info* section lists the command you need to run to get the autogenerated password. Run the command to get your admin password.
-. Navigate to *Networking* -> *Routes*.
-. Find the *central* Route and click on the RHACS portal link under the *Location* column.
-. Log in to the RHACS portal using the username *admin* and the autogenerated password.
+. The *Admin Credentials Info* section lists the command you need to run to get the autogenerated password.
+    . Use the {ocp} CLI to enter the command to get your admin password:
```

Comment:
Was this meant to be a substep? Wasn't sure if the white space was intentional. If it's a substep, try `..` instead.